model/gitlab/gitlab.go

```diff
@@ -46,7 +46,6 @@ func userFromGitLabUser(glu *GitLabUser) *model.User {
 	} else {
 		user.FirstName = glu.Name
 	}
-	strings.TrimSpace(user.Email)
```

Comment:
`TrimSpace` is a pure function; it doesn't do anything if we ignore the return value. So clearly we don't actually need this line here...